lib/recommended-rules.js

```diff
@@ -73,6 +73,7 @@ module.exports = {
   "ember/require-tagless-components": "error",
   "ember/require-valid-css-selector-in-test-helpers": "error",
   "ember/routes-segments-snake-case": "error",
+  "ember/template-no-let-reference": "error",
```

Comment:
Can we have a separate PR to enable any rules as recommended? It's more clear what's going on then and will be better for the resulting changelog. Normally, we never create rules at the same time as enabling them as recommended.